File: Baekjoon/C++/1330. 두 수 비교하기/solution.cpp
// https://www.acmicpc.net/problem/1330

#include <iostream>

using namespace std;

int main(int argc, char const *argv[])
{
    int a, b;
    cin >> a >> b;
    if (a > b)
        cout << ">";
    else if (a == b)
        cout << "==";
    else
        cout << "<";
}

File: Baekjoon/Go/1330. 두 수 비교하기/solution.go
// https://www.acmicpc.net/problem/1330

package main

import "fmt"

func main() {
	var a, b int
	fmt.Scan(&a, &b)
	if a > b {
		println(">")
	} else if a == b {
		println("==")
	} else {
		println("<")
	}
}
